internal/proxy/hook/authz/authz.go

```diff
@@ -194,16 +202,26 @@ func (a Authz) ServeHook(res *http.Response, err error) (*http.Response, error)
 	return a.next.ServeHook(res, nil)
 }
 
-func createResources(permissionAttributes map[string]interface{}) ([]resource.Resource, error) {
+func createResources(ctx context.Context, permissionAttributes map[string]interface{}, p ProjectService) ([]resource.Resource, error) {
 	var resources []resource.Resource
 	projects, err := getAttributesValues(permissionAttributes["project"])
 	if err != nil {
 		return nil, err
 	}
 
```

Comment:
Is it really needed to pass the project service as argument? If not, better to do this
```suggestion
func (a Authz) createResources(ctx context.Context, permissionAttributes map[string]interface{}) ([]resource.Resource, error) {
	var resources []resource.Resource
	projects, err := getAttributesValues(permissionAttributes["project"])
	if err != nil {
		return nil, err
	}
```